Content.Shared/Xenoarchaeology/Equipment/Components/ArtifactAnalyzerComponent.cs

```diff
@@ -16,6 +16,18 @@ public sealed partial class ArtifactAnalyzerComponent : Component
     [DataField]
     public TimeSpan AnalysisDuration = TimeSpan.FromSeconds(30);
 
+    // Begin DeltaV - Tie point output to glimmer
+    /// <summary>
+    /// The ratio of research points per one glimmer.
+    /// </summary>
+    public int ExtractRatio = 750;
+
+    /// <summary>
+    /// The maximum added multiplier, reached at max glimmer.
+    /// </summary>
+    [DataField]
+    public float PointGlimmerMultiplier = 8f;
+    // End DeltaV - Tie point output to glimmer
```

Comment:
I was recently instructed that we can just put DeltaV at the start of the summaries!

```suggestion
    /// <summary>
    /// DeltaV - The ratio of research points per one glimmer.
    /// </summary>
    public int ExtractRatio = 750;

    /// <summary>
    /// DeltaV - The maximum added multiplier, reached at max glimmer.
    /// </summary>
    [DataField]
    public float PointGlimmerMultiplier = 8f;
```